torch/csrc/lazy/core/ir_builder.h

```diff
@@ -266,5 +267,21 @@ static inline NodePtr MakeSizeDiv(const Value& a, const Value& b) {
   return getIrBuilder()->MakeSizeDiv(a, b);
 }
 
+inline Value GetSymIntValue(c10::SymInt a) {
```

Comment:
Yes, it is intentionally incongruent, because I don't know how to represent a list of symints in LTC IR, there is not support for this. TensorList has a similar problem in LTC and it is worked around in a way that I don't understand.